### PR TITLE
CO_Person_Role_country_attribute

### PR DESCRIPTION
**Pull request by ioigoume**
Add Country ISO codes.
Allow country field as the only address input during registration.

### CODE REVIEW

File: app/Lib/lang.php [hunk 1]
```diff
@@ -660,8 +660,261 @@
                             RequiredEnum::NotPermitted => 'Not Permitted'),
   
   'en.required.address' => array(RequiredAddressFieldsEnum::Street                       => 'Street',
+                                 RequiredAddressFieldsEnum::Country                      => 'Country',
                                  RequiredAddressFieldsEnum::StreetCityStatePostal        => 'Street, City, State, ZIP/Postal Code',
                                  RequiredAddressFieldsEnum::StreetCityStatePostalCountry => 'Street, City, State, ZIP/Postal Code, Country'),
+
+  'en.country.codes' => array(
+    "AF" => "Afghanistan",
+    "AX" => "Ã…land Islands",
```
Comment (lionick) [inline app/Lib/lang.php]: Is this the correct name?

File: app/Lib/lang.php [hunk 2]
```diff
@@ -660,8 +660,261 @@
...
+    "CN" => "China",
+    "CX" => "Christmas Island",
+    "CC" => "Cocos (Keeling) Islands",
+    "CO" => "Colombia",
+    "KM" => "Comoros",
+    "CG" => "Congo",
+    "CD" => "Congo, the Democratic Republic of the",
+    "CK" => "Cook Islands",
+    "CR" => "Costa Rica",
+    "CI" => "CÃ´te d'Ivoire",
```
Comment (lionick) [inline app/Lib/lang.php]: Is this the correct name?

**Review by lionick — CHANGES_REQUESTED** [review]

File: app/Lib/lang.php (hunk 2 above)
Comment (ioigoume) [reply app/Lib/lang.php]: Done

**Review by lionick — APPROVED** [review]
LGTM

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**